docs/dev/table/tableApi.md

```diff
@@ -1682,6 +1682,36 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
 {% top %}
 
+### Aggregate
```

Comment:
Users may be confused to see `Aggregations` and `Aggregate`. 
Should we divide the current `Operations` chapter into tow sub ones?
- SQL base Operations
- Non SQL base Operations

And we can put all map/flatmap/aggregate/flataggregate into the `Non SQL base Operations` chapter. It is more clear.

Reply:
Good suggestion.